pkg/virtctl/create/vm/vm_test.go

```diff
@@ -278,6 +283,54 @@ var _ = Describe("create vm", func() {
 			Entry("with namespace, name, size and bootorder", "my-ns", "my-dv", "my-dvt", "10Gi", 8, "src:my-ns/my-dv,name:my-dvt,size:10Gi,bootorder:8"),
 		)
 
+		DescribeTable("VM with specified volume source", func(params, source string) {
+			out, err := runCmd(setFlag(VolumeImportFlag, params))
+			Expect(err).ToNot(HaveOccurred())
+			vm := unmarshalVM(out)
+
+			Expect(vm.Spec.DataVolumeTemplates[0].Spec.Storage.Resources.Requests[k8sv1.ResourceStorage]).To(Equal(resource.MustParse(size)))
+
+			switch source {
+			case Blank:
+				Expect(*vm.Spec.DataVolumeTemplates[0].Spec.Source.Blank).ToNot(BeNil())
```

Comment:
Apologies there was a typo in my previous comment, you don't need to de-reference `vm.Spec.DataVolumeTemplates[0].Spec.Source.Blank` here.